packages/editor/src/lib/editor/tools/BaseBoxShapeTool/children/Pointing.ts

```diff
@@ -132,3 +134,10 @@ export class Pointing extends StateNode {
 		this.parent.transition('idle')
 	}
 }
+
+function maybeSnapToGrid(point: Vec, editor: Editor): Vec {
```

Comment:
duplicated from `shapes.ts`? let's have this function here be the canonical function, then you can export it as `@public` and use it elsewhere in the library

Reply:
Yes! Thanks!